cis/gcp/compute/gcp-cis-4.4-compute-enable-connecting-to-serial-ports-is-not-enabled-for-vm-instance/test/gcp-cis-4.4-compute-enable-connecting-to-serial-ports-is-not-enabled-for-vm-instance/failure-null-metadata.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This Sentinel configuration file should not be required if you implement my suggestion.